ts/packages/cli/src/helpers/synchronizers/logic/GcpCloudFunctions.ts

```diff
@@ -0,0 +1,32 @@
+import { CloudFunctionsServiceClient } from '@google-cloud/functions';
+import { EvaluatedConfigs } from '@configu/ts';
+import { CONFIG_SYNCHRONIZER_LABEL } from '@configu/lib';
+
+const label = CONFIG_SYNCHRONIZER_LABEL.GcpCloudFunctions;
+
+export type GcpCloudFunctionsConfiguration = {
+  // * GCP Cloud Functions docs: https://cloud.google.com/functions/docs/configuring/env-var
+  keyFile: string;
+  functionName: string;
+};
+
+// TODO - Decide between syncGcpCloudFunctionsConfigs (previous name) and syncGcpCloudFunctionsEnvVars
+export const syncGcpCloudFunctionsEnvVars = async ({
```

Comment:
@rannn505 We previously named this `syncGcpCloudFunctionsConfigs`, GCP cloud functions names their 'configs' as 'runtime environment variable' so I thought `syncGcpCloudFunctionsEnvVars` would be more appropriate here. Opinion?

Reply:
We decided on `syncGcpCloudFunctionsEnvironmentVariables`.